cmd/mailbox-controller/main.go

```diff
@@ -109,20 +115,29 @@ func main() {
 	inventoryClientConfig.UserAgent = "mailbox-controller"
 
 	// Get client config for view of SyncTarget objects
-	syncTargetViewConfig, err := configForViewOfExport(ctx, inventoryClientConfig, "workload.kcp.io")
+	//syncTargetViewConfig, err := configForViewOfExport(ctx, inventoryClientConfig, "workload.kcp.io")
+	syncTargetViewConfig, err := configForViewOfExport(ctx, inventoryClientConfig, "edge.kcp.io")
 	if err != nil {
 		logger.Error(err, "Failed to create client config for view of SyncTarget exports")
 		os.Exit(4)
 	}
 
-	stViewClusterClientset, err := kcpclusterclientset.NewForConfig(syncTargetViewConfig)
+	//stViewClusterClientset, err := kcpclusterclientset.NewForConfig(syncTargetViewConfig)
+	//if err != nil {
+	//	logger.Error(err, "Failed to create clientset for view of SyncTarget exports")
+	//	os.Exit(6)
+	//}
```

Comment:
Why did you comment out and wrote the code again below ?

Reply:
There is a package name change in the retained line.  I do not understand the `if` statement.

Reply:
just to be able to restore if testing didn't succeed

Reply:
This is fine, but once testing pass (before the PR) I would remove all old code

Reply:
we can always see the old code by a simple diff